### PR TITLE
Install default storageclass in AWS & GCE envs

### PR DESCRIPTION
**Pull request by gnufied**
Used most of the code from - https://github.com/openshift/openshift-tools

I have tested it to work (manually). I removed the quota support from the role but that can be added back, if required.

Fixes https://trello.com/c/RTTIC2SU/431-5-openshift-installer-and-default-storageclass

### CODE REVIEW

**Comment by gnufied** [comment]
aos-ci-test

**Comment by gnufied** [comment]
@sdodson can you help with review please?

File: roles/openshift_default_storage_class/meta/main.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  description: This role configures the StorageClass in Openshift
+  company: Red Hat
+  license: Apache
+  min_ansible_version: 2.2
+  platforms:
+  - name: EL
+    verisons:
+    - 7
+  categories:
+  - cloud
```
Comment (sdodson) [inline roles/openshift_default_storage_class/meta/main.yml]: I think you need to add a dependency on lib_openshift here. like this, though without the other two dependencies

https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_storage_glusterfs/meta/main.yml#L12-L15

Comment (gnufied) [inline roles/openshift_default_storage_class/meta/main.yml]: done

**Comment by childsb** [comment]
@sdodson has everything been addressed, and are we OK to merge now?

**Comment by sdodson** [comment]
aos-ci-test

**Comment by sdodson** [comment]
@childsb I'm fine with it, just not sure if this qualifies for merging during devcut or not. Your call.

**Comment by childsb** [comment]
I'll hold merge for 3.7 unless QE opens a BZ

**Comment by sdodson** [comment]
github api rate limits

**Comment by sdodson** [comment]
aos-ci-test

**Comment by sdodson** [comment]
[merge]

**Comment by sdodson** [comment]
This is what's being requested in this BZ? correct https://github.com/openshift/openshift-ansible/pull/4262/files

**Comment by sdodson** [comment]
[merge]

**Pull request merged**